### PR TITLE
feat: add bin program

### LINKED ISSUE #24: Publish package to NPM
I want to quickly try out this importer.

Currently, I'd have to clone the project, install dependencies, and then run `ts-node` (which first compiles the source code). It would be easier for new users if they could run this in a terminal:

```bash
$ npm i -g linear-import # or yarn global add linear-import
$ linear-import
```

For this, there'd need to be a `bin` entry added to `package.json` and the package should be published to NPM.

Are you open for a PR to implement the `bin` entry? ([prepared a branch here](https://github.com/linearapp/linear-import/compare/master...hongaar:feat/bin?expand=1))
Obviously, publishing should be taken care of by the maintainers.

### PR DESCRIPTION
**Pull request by hongaar**
By adding a bin entry to package.json and changing the build entry to cli.js, users can install the CLI (globally) without having to clone this repository.

```bash
$ npm i -g linear-import # or yarn global add linear-import
$ linear-import
```

For this to work, this package would also need to be published to NPM.

Closes #24

### CODE REVIEW

**Review by rlargman — COMMENTED** [review]
Thank you for the PR! This is a great idea, but we don't want to take on publishing the package to NPM at this moment. However, if we go down that route, being able to install the CLI without the hassle of cloning the repository is something we would want to do!

**Review by rlargman — COMMENTED** [review]
Thank you for the PR! This is a great idea, but we don't want to take on publishing the package to NPM at this moment. However, if we go down that route, being able to install the CLI without the hassle of cloning the repository is something we would want to do!

**Comment by hongaar** [comment]
> [...] we don't want to take on publishing the package to NPM at this moment.

Any particular reason for this? If it's workflow related, I can prepare a PR with a GitHub Action to test, build and publish this repo. Let me know!

**Comment by rlargman** [comment]
Thanks for the offer, that would be great! I've published the initial version to NPM so far.

File: README.md [hunk 1]
```diff
@@ -79,5 +79,6 @@ Following fields are supported:
 ## Todo
 
 - [x] Automatic image uploads
+- [ ] Publish to NPM
```
Comment (rlargman) [inline README.md:82]:
```suggestion
```

**Pull request merged**